contributing/adding_datasets.md

```diff
@@ -9,232 +9,109 @@ parent: Contributing to Data Commons
 
 ```
-Node: E:COVIDTracking_States->E0
-typeOf: dcs:StatVarObservation
-variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19
-measurementMethod: dcs:CovidTrackingProject
-observationAbout: C:COVIDTracking_States->GeoId
-observationDate: C:COVIDTracking_States->Dates
-value: C:COVIDTracking_States->CumulativeCount_MedicalTest_COVID_19
-
-Node: E:COVIDTracking_States->E1
+Node: l:MNHOWN_E1_R2
 typeOf: dcs:StatVarObservation
-variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19_Positive
-measurementMethod: dcs:CovidTrackingProject
-observationAbout: C:COVIDTracking_States->GeoId
-observationDate: C:COVIDTracking_States->Dates
-value: C:COVIDTracking_States->CumulativeCount_MedicalTest_COVID_19_Positive
-
-... etc ...
-```
-
-The mapping for the remaining variables is essentially identical and can be generated by a script.
-
-**Note:** This example adds the property `measurementMethod: dcs:CovidTrackingProject` to distinguish the same CumulativeCount_MedicalTest_COVID_19 observation for a state and date from multiple sources (such as the New York Times, the COVID Tracking Project, or Harvard).
-
+variableMeasured: dcid:Count_HousingUnit_OwnerOccupied_AsFractionOf_Count_HousingUnit_OccupiedHousingUnit
+observationAbout: l:MNHOWN_E0_R2
+observationDate: 1985-01-01
+value: 70.0
 ```
-# Automate Template MCF generation since there are many Statistical Variables.
-TEMPLATE_MCF_TEMPLATE = """
-Node: E:COVIDTracking_States->E{index}
-typeOf: dcs:StatVarObservation
-variableMeasured: dcs:{stat_var}
-measurementMethod: dcs:CovidTrackingProject
-observationAbout: C:COVIDTracking_States->GeoId
-observationDate: C:COVIDTracking_States->Dates
-value: C:COVIDTracking_States->{stat_var}
-"""
-```
-
-Check in the Template MCF file together with the cleaned CSV and its preprocessing script to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data). The Template MCF generation code may also be included (as in this example).
-
-## Conclusion
 
-After producing all three required components (`StatisticalVariable` schema MCF, cleaned CSV, and Template MCF), go ahead and submit a PR to [https://github.com/datacommonsorg/data/pulls](https://github.com/datacommonsorg/data/pulls).
\ No newline at end of file
+Once you've generated the MCF, write a README following the template at <https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset>. Finally, check in the MCF file together with any script used to assemble it and the README to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data).
```

Comment:
Maybe use this link for second link: https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset